### PR TITLE
fix poorly rendered nested selectors

### PR DESCRIPTION
**Pull request by lubegasimon**

### CODE REVIEW

**Comment by davesnx** [comment]
I expected the following tests to be there:

```
.one_level { &.without_space { color: blue } };

.one_level { & .with_space { color: blue } };

.one_level { &.$(with_interpolation) { color: blue } };

.one_level { & .$(space_with_interpolation) { color: blue } };
```

```
.two_levels { & .first_level { & .second_level { color: blue } } };

...
```

```
.without_ampersand { .one_level { color: blue } }; // expects an space here
```

```
.$(interpolation_at_the_start) { & .one_level { color: blue } };
```

```
#using_ids { & .one_level { color: blue } };
```

```
[using_attrs] { & .one_level { color: blue } };
```

**Comment by lubegasimon** [comment]
> I expected the following tests to be there:
> ...

Thank you, I am happy to add them

File: packages/parser/lib/css_resolver.re [hunk 1]
```diff
@@ -0,0 +1,164 @@
+let is_nested = ref(false);
```
Comment (lubegasimon) [inline packages/parser/lib/css_resolver.re:1]: I am not convinced by this at all!

File: packages/parser/lib/css_resolver.re [hunk 2]
```diff
@@ -0,0 +1,164 @@
...
+
+let resolve_subclass_selector = (subclass: Css_types.subclass_selector) => {
+  Css_types.(
+    switch (subclass) {
+    | Id(id) when ! is_nested^ => Id(" #" ++ id)
+    | Class(c) when ! is_nested^ => Class(" ." ++ c)
+    | ClassVariable(vars) when ! is_nested^ =>
+      ClassVariable(vars |> List.map(var => " ." ++ var))
```
Comment (lubegasimon) [inline packages/parser/lib/css_resolver.re:9]: To be reworked, this doesn't yield a property rendering.

**Pull request closed without merging**